### PR TITLE
Update FRS docs to match the latest client API

### PR DESCRIPTION
**Pull request by sumedhb1995**
0.47 had a number of changes to the `AzureClient` API. This updates the doc to match those changes along with fixing a number of grammatical, spacing, and other nit issues

### CODE REVIEW

File: docs/content/docs/deployment/azure-frs.md [hunk 1]
```diff
@@ -20,126 +17,154 @@ To connect to an Azure Fluid Relay instance you first need to create an `AzureCl
...
-    name: "my-container",
-    initialObjects: {
-        /* ... */
-    },
-    dynamicObjectTypes: [ /*...*/ ],
-}
-const azureClient = new AzureClient(config);
-await azureClient.createContainer({ id: "_unique-id_" }, schema);
-const { fluidContainer, containerServices } = await azureClient.getContainer({ id: "_unique-id_" }, schema);
+  name: "my-container",
```
Comment (micahgodbolt) [inline docs/content/docs/deployment/azure-frs.md:93]: name was removed from schema, right?

File: docs/content/docs/deployment/azure-frs.md [hunk 2]
```diff
@@ -20,126 +17,154 @@ To connect to an Azure Fluid Relay instance you first need to create an `AzureCl
...
+  initialObjects: {
+    /* ... */
+  },
+  dynamicObjectTypes: [
+    /*...*/
+  ],
+};
+const azureClient = new AzureClient(clientProps);
+const { fluidContainer, containerServices } = await azureClient.createContainer(
```
Comment (micahgodbolt) [inline docs/content/docs/deployment/azure-frs.md:102]:
```suggestion
const { container, services } = await azureClient.createContainer(
```

File: docs/content/docs/deployment/azure-frs.md [hunk 3]
```diff
@@ -20,126 +17,154 @@ To connect to an Azure Fluid Relay instance you first need to create an `AzureCl
...
+  schema
+);
+const id = await fluidContainer.attach();
```
Comment (micahgodbolt) [inline docs/content/docs/deployment/azure-frs.md:105]:
```suggestion
const id = await container.attach();
```

File: docs/content/docs/deployment/azure-frs.md [hunk 4]
```diff
@@ -20,126 +17,154 @@ To connect to an Azure Fluid Relay instance you first need to create an `AzureCl
...
 ```
 
-The `id` being passed into the container config is a unique identifier to a container instance. Any client that wants to join the same collaborative session needs to call `getContainer` with the same container `id`.
+The `container.attach()` call is when the container actually becomes connected to the service and is recorded in its blob storage. It returns an `id` which is the unique identifier to this container instance.
+
+Any client that wants to join the same collaborative session needs to call `getContainer` with the same container `id`.
+
+```javascript
+const { fluidContainer, containerServices } = await azureClient.getContainer(
```
Comment (micahgodbolt) [inline docs/content/docs/deployment/azure-frs.md:113]:
```suggestion
const { container, services } = await azureClient.getContainer(
```

File: docs/content/docs/deployment/azure-frs.md [hunk 5]
```diff
@@ -20,126 +17,154 @@ To connect to an Azure Fluid Relay instance you first need to create an `AzureCl
...
+```
 
 For the further information on how to start recording logs being emitted by Fluid, see [Telemetry]({{< relref "telemetry.md" >}}).
 
 The container being fetched back will hold the `initialObjects` as defined in the container schema. See [Data modeling]({{< relref "data-modeling.md" >}}) to learn more about how to establish the schema and use the `FluidContainer` object.
 
 ## Getting audience details
 
-Calls to `createContainer` and `getContainer` return an `AzureResources` object that contains a `FluidContainer` -- described above -- and a `containerServices` object.
+Calls to `createContainer` and `getContainer` return two values: a `FluidContainer` -- described above -- and a `containerServices` object.
```
Comment (micahgodbolt) [inline docs/content/docs/deployment/azure-frs.md:125]:
```suggestion
Calls to `createContainer` and `getContainer` return two values: a `container` -- described above -- and a `services` object.
```

File: docs/content/docs/deployment/azure-frs.md [hunk 6]
```diff
@@ -20,126 +17,154 @@ To connect to an Azure Fluid Relay instance you first need to create an `AzureCl
...
 
-The `FluidContainer` contains the Fluid data model and is service-agnostic. Any code you write against this container object returned by the `FrsClient` is reusable with the client for another service. An example of this is if you prototyped your scenario using `TinyliciousClient`, then all of your code interacting with the shared objects within the Fluid container can be reused when moving to using `FrsClient`.
+The `FluidContainer` contains the Fluid data model and is service-agnostic. Any code you write against this container object returned by the `AzureClient` is reusable with the client for another service. An example of this is if you prototyped your scenario using `TinyliciousClient`, then all of your code interacting with the shared objects within the Fluid container can be reused when moving to using `AzureClient`.
```
Comment (micahgodbolt) [inline docs/content/docs/deployment/azure-frs.md:127]:
```suggestion
The `container` contains the Fluid data model and is service-agnostic. Any code you write against this container object returned by the `AzureClient` is reusable with the client for another service. An example of this is if you prototyped your scenario using `TinyliciousClient`, then all of your code interacting with the shared objects within the Fluid container can be reused when moving to using `AzureClient`.
```

File: docs/content/docs/deployment/azure-frs.md (hunk 1 above)
Comment (timtwang) [reply docs/content/docs/deployment/azure-frs.md]: yup! It's just `initialObjects` and `dynamicObjectTypes` now

**Review by micahgodbolt — APPROVED** [review]

**Pull request merged**